packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5980,6 +5981,7 @@ class RawImage extends LeafRenderObjectWidget {
       invertColors: invertColors,
       isAntiAlias: isAntiAlias,
       filterQuality: filterQuality,
+      devicePixelRatio: View.maybeOf(context)?.devicePixelRatio,
```

Comment:
note to self: here and below use `MediaQuery.maybeDevicePixelRatioOf(context)` instead.